nbinteract/config.py

```diff
@@ -79,9 +84,6 @@ class ProductionConfig(Config):
     # URL for users to access. Make sure it has a trailing slash.
     URL = os.environ.get('JPY_BASE_URL', default='') + "/"
```

Comment:
I left that in there just in case someone wanted to still use it. But I just realized now that I was incorrect in thinking that leaving it will still keep that functionality available. The value of `URL` will always be overwritten by the `base_url` constructor argument for `Config`.

I'll just remove it for now.